WSLDiskShrinker/Common/BooleanToVisibilityConverter.cs

```diff
@@ -5,15 +5,13 @@ namespace WSLDiskShrinker.Common;
 
 class BooleanToVisibilityConverter : IValueConverter
 {
-	public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
-	{
-		bool v = (bool)value;
-		return v ? Visibility.Visible : Visibility.Collapsed;
-	}
+    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        return (bool)value ? Visibility.Visible : Visibility.Collapsed;
+    }
 
-	public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
-	{
-		throw new NotImplementedException();
-	}
+    public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        return (bool)value ? Visibility.Collapsed : Visibility.Visible; // Not used.
+    }
```

Comment:
The same issue here. Wrong implementation here. And definitely no need to implement it.